docker-compose/config/cse/d2cc/cse-d2cc-gridcapa-job-launcher.yml

```diff
@@ -14,16 +14,17 @@ spring:
     stream:
       bindings:
         run-task:
-          destination: cse-d2cc-run-task
+          destination: cse-d2cc-launch-task
           group: cse-d2cc-job-launcher
 
 job-launcher:
   url:
     task-manager-timestamp-url: http://cse-d2cc-gridcapa-task-manager:8080/tasks/
```

Comment:
the host should match the name of the service in the docker-compose file, here cse-d2cc-task-manager